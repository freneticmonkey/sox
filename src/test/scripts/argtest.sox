
fn main(argc, argv) {
    print("arg count is: ")
    print(argc)
    print("argv is ")
    print(argv)
}