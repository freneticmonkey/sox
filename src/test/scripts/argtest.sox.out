arg count is: 
2
argv is 
[sox,src/test/scripts/argtest.sox]
